itsnameissteven/Refactor-Tractor-Fitlit
# Feature/post requests

## Pull Request

### Description
* Is this a feature or a fix?

This is a feature.

* Where should the reviewer start?

Start in the `scripts.js` file, look at the function `handlePostRequest(link, body)` then in the `API.js` file look at the `addNewData(link, newData)` method which is a dynamic POST request method.

* What functionalities do these changes effect? Why were these changes made? What was the motivation behind these changes?

Changes were described above ^. User is able to input hydration, sleep, and activity information and their new information is displayed on the DOM.

* What outside features or research did you use to implement this fix?

Lessons.

* How should this be tested?

Load the website locally and try to input new activity/sleep/hydration information, page should update.

### Relevant Screen Shots
***
#### Please review considerations below:
